src/bin/tikv-server.rs

```diff
@@ -28,6 +28,7 @@ extern crate signal;
 #[cfg(unix)]
 extern crate nix;
 extern crate prometheus;
+extern crate sysinfo;
```

Comment:
It only supports Linux and macOS, how about other platforms?
https://github.com/GuillaumeGomez/sysinfo#system-